internal/packages/consensus/uptime/collector/collector.go

```diff
@@ -17,9 +17,6 @@ var (
 	_ common.CollectorLoop  = loop
 )
 
-// NOTE: this is for solo mode
```

Comment:
the global `packageMonikers` variable needed to be removed in all packages as it created race conditions due to multiple loops writing to this variable at the same time. This worked as long as the `moniker` variable was the same for all loops, but with my added change which allows moniker configuration per chain, this creates unexpected results. 
As the config is passed to the loop via the `p` variable anyway, `p.Monikers` is used instead, removing the race-condition